src/components/preview.tsx

```diff
@@ -178,7 +178,7 @@ export const Preview: Component<Props> = (props) => {
   // We have that the iframe src is entangled with its loading state
   const iframeLoader = createMemo(() => {
     const html = generateHTML(props.isDark, JSON.stringify({ imports: props.importMap }));
-    const loaded = createSignal(false);
+    const loaded = createSignal(false, { equals: false });
```

Comment:
why this change?

Reply:
> why this change? I'd expect that on reload the signal should go to false and should only get sent when it goes back to true?

As far as I can tell, there's no way to listen for when an iframe reloads